packages/react-log-viewer/src/LogViewer/examples/Logger.md

```diff
@@ -0,0 +1,64 @@
+---
+id: Logger
```

Comment:
you should change these reference and the file name to LogViewer

Reply:
or remove them? what is the difference between the two markdown files you have here?